opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -158,8 +158,7 @@ message Span {
   // This field is semantically required and it is expected that end_time >= start_time.
   fixed64 end_time_unix_nano = 8;
 
-  // attributes is a collection of key/value pairs. The value can be a string,
-  // an integer, a double or the Boolean values `true` or `false`. Note, global attributes
+  // attributes is a collection of key/value pairs. Note, global attributes
```

Comment:
The attribute in common.proto allows more than the OTel trace API spec. Not sure if this restriction applies to OTLP in general if used without OTel theoretically. But in doubt I would point to https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/common.md#attributes for the allowed subset.

Reply:
Good point, I added a reference.